app/Models/CountriesPlaceBySports.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class CountriesPlaceBySports extends Model
```

Comment:
@hayaIshida are you sure you needed a separate pivot Model class here? You could filter the pivot table with Laravel functions, even without having this model.

Reply:
@PovilasKorop  I used this models upsert function in the controller, but probably there are nicer ways to persist the relationship data.
Thank you for the review!